pkg/operators/manager.go

```diff
@@ -51,7 +50,7 @@ func RunClusterManagerOperator(ctx context.Context, controllerContext *controlle
 		return err
 	}
 
-	kubeInformer := informers.NewSharedInformerFactoryWithOptions(kubeClient, 5*time.Minute, informers.WithNamespace(helpers.ClusterManagerNamespace))
```

Comment:
add a todo here for potential performance impact.

Reply:
Comment added.